### ISSUE TITLE
Java 14 support: gmavenplus-plugin likely not compatible

### ISSUE THREAD

**Issue by orpiske** [apache/camel-kafka-connector#910]
Trying to build in CI with JDK 14 causes this issue:

```
[INFO] Building Camel-Kafka-Connector :: Buildtools 0.8.0-SNAPSHOT        [3/3]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:3.1.0:clean (default-clean) @ camel-buildtools ---
[INFO] 
[INFO] --- maven-enforcer-plugin:1.3.1:enforce (enforce-maven-version) @ camel-buildtools ---
[INFO] 
[INFO] --- maven-enforcer-plugin:1.3.1:enforce (enforce-java) @ camel-buildtools ---
[INFO] 
[INFO] --- maven-remote-resources-plugin:1.6.0:process (process-resource-bundles) @ camel-buildtools ---
[INFO] Skipping remote resources execution.
[INFO] 
[INFO] --- license-maven-plugin:3.0:format (default-cli) @ camel-buildtools ---
[INFO] Updating license headers...
[INFO] 
[INFO] --- gmavenplus-plugin:1.6.2:execute (default) @ camel-buildtools ---
java.lang.NoClassDefFoundError: Could not initialize class org.codehaus.groovy.vmplugin.v7.Java7
```

**Comment by apupier** [comment]
needs to upgrade to at least version 1.9.0 https://github.com/groovy/GMavenPlus/releases/tag/1.9.0

**Comment by orpiske** [comment]
Should be fixed with #918
